src/linear_solver_routines/block_col_projection.jl

```diff
@@ -0,0 +1,53 @@
+# This file is pat of RLinearAlgebra.jl
+"""
+    LinSysBlkColGent <: LinSysBlkColProjection
+
+A mutable structure that represents a standard block column projection method.
+
+# Aliases
+- `BlockCoordinateDescent`
+
+# Fields
+- `alpha::Float64`, a relaxation parameter that should be set between `0.0` and `2.0`
```

Comment:
The docstring symbol and the structure symbol should be the same.

Reply:
I have corrected this.